js/data.js

```diff
@@ -0,0 +1,83 @@
+import { getRandomFloat } from './util.js';
+import { getRandomInt } from './util.js';
+import { getArrayElement } from './util.js';
+import { getNewArray } from './util.js';
```

Comment:
Если импортируем из одного файла то через запятую. Если получается очень длинно то тогда расополагаем в столбик. С экспортами аналогично
```suggestion
import { getRandomFloat, getRandomInt ....... } from './util.js';
```